scikeras/wrappers.py

```diff
@@ -474,84 +474,88 @@ def _check_array_dtype(arr):
             y = check_array(
                 y, ensure_2d=False, allow_nd=False, dtype=_check_array_dtype(y)
             )
+            y_dtype_ = y.dtype
+            y_ndim_ = y.ndim
+            if reset:
+                self.target_type_ = type_of_target(y)
```

Comment:
Is this necessary to define?

Reply:
I think it's an important attribute of `y` that can be useful in building the model. We use it in `dynamic_classifier`. I imagine users may want to as well. I think there's no other way to transmit that information that succinctly.

Reply:
I think you're right; it's not a huge performance burden for two reasons: it's done on `y` (not the much larger `X`) and is only done once per fit. I'm resolving this thread.